src/cowrie/output/datadog.py

```diff
@@ -0,0 +1,63 @@
+"""
+Simple Datadog HTTP logger.
+"""
+
+from __future__ import annotations
+
+import json
+import platform
+
+from io import BytesIO
+from twisted.internet import reactor
+from twisted.internet.ssl import ClientContextFactory
+from twisted.web import client, http_headers
+from twisted.web.client import FileBodyProducer
+
+import cowrie.core.output
+from cowrie.core.config import CowrieConfig
+
+
+class Output(cowrie.core.output.Output):
+    def start(self):
+        self.url = CowrieConfig.get("output_datadog", "url").encode("utf8")
```

Comment:
`output_datadog` would benefit from being extracted in a constant as it's repeated several times

Reply:
Agree. I pick inspiration from other existing modules.

Reply:
There's no performance issue. So I let the config getter this way.